File: assets/samples/email.verification/perl/email_api.pl
#!/usr/bin/perl

use LWP::Simple;                # From CPAN
use JSON qw( decode_json );     # From CPAN
use Data::Dumper;
use strict;
use warnings;

my $email = "support@whoisxmlapi.com";
my $api_key = "your_api_key";
my $api_url = "https://emailverification.whoisxmlapi.com/api/v1";

my $url = "$api_url?apiKey=$api_key&emailAddress=$email";

getprint($url);

File: assets/samples/email.verification/nodejs/email_api.js
var http = require('http');

var email = 'support@whoisxmlapi.com';
var api_key = 'your_api_key';
var api_url = 'https://emailverification.whoisxmlapi.com/api/v1?';

var url = api_url + 'apiKey=' + api_key + '&emailAddress=' + email;

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();
